### PR TITLE
Queue mediation

### PR DESCRIPTION
**Pull request by ayan-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -64,7 +64,7 @@ resources:
             Value:
               Ref: AqtsCaptureTriggerLambdaFunction
         MetricName: ConcurrentExecutions
-        Statistic: Sum
+        Statistic: Maximum
```
Comment (ayan-usgs) [inline serverless.yml:68]: This is actually more useful than the `Sum` statistic.

File: serverless.yml [hunk 2]
```diff
@@ -91,25 +91,51 @@ resources:
...
+                      Value: .json
+              Queue:
+                Fn::GetAtt:
+                  - triggerQueue
+                  - Arn
+    triggerQueue:
+      Type: AWS::SQS::Queue
+      Properties:
+        QueueName: aqts-capture-trigger-queue-${self:provider.stage}
+        MessageRetentionPeriod: 172800
```
Comment (ssoper-usgs) [inline serverless.yml]: Just out of curiosity - what is the significance of the `172800` duration.  From the doc it looks like that is the number of seconds to retain a message or ~ 2 days.

**Review by ssoper-usgs — APPROVED** [review]
Nice tests!

File: serverless.yml (hunk 2 above)
Comment (ayan-usgs) [reply serverless.yml]: I wanted to messages to last at least a long as the files in S3 (1 - 2 days).

Comment (ssoper-usgs) [reply serverless.yml]: Roger that - thanks!

Comment (dsteinich) [reply serverless.yml]: Should we consider a longer retention on prod? Enough to allow us to fix errors that happen on a long weekend?

Comment (ayan-usgs) [reply serverless.yml]: That would probably be a could idea. The retention in the S3 bucket would need to be increased to match so the initial data would be kept too.

Comment (ayan-usgs) [reply serverless.yml]: I've increased it to four days.

**Pull request merged**